README.md

```diff
@@ -56,7 +94,7 @@ And then fire up the environment and use the provided Makefile targets to do
 common tasks:
 
 ```console
-$ docker run -it -v $PWD:/src -w /src edgecontext
+$ docker run -it -v $PWD:/src --user "$(id -u):$(id -g)" -w /src edgecontext
```

Comment:
cc @spladug: I added `--user` to avoid getting files owned by root in the working directory.

Reply:
sgtm